common/history/characters/0_dvg_shekhbizin .txt

```diff
@@ -0,0 +1,35 @@
+﻿CHARACTERS = {
+	c:SHZ = {
+	    create_character = {
```

Comment:
The ruler of Shekhbizin Empire is scripted to be Sadeq Shekhbizin but that character leads the interest group of Qashqai Confederacy that they belong to, which is not allowed under Dictatorial/Hereditary